src/main/java/com/mig82/folders/wrappers/ParentFolderBuildWrapper.java

```diff
@@ -26,107 +25,52 @@
  *
  * @author Miguelangel Fernandez Mendoza
  */
-public class ParentFolderBuildWrapper extends SimpleBuildWrapper{
```

Comment:
Seems you've changed all the indentation from tabs to spaces, so it looks like the whole file is changed. Please change the indentation back to tabs, so I can clearly see what you've removed/added.